meta-lmp-base/recipes-containers/containerd/containerd-opencontainers/0001-Makefile-allow-GO_BUILD_FLAGS-to-be-externally-speci.patch

```diff
@@ -1,35 +0,0 @@
-From 474b93f8f643651144304a6971ef476add7f06f8 Mon Sep 17 00:00:00 2001
-From: Bruce Ashfield <bruce.ashfield@gmail.com>
```

Comment:
Same.

Reply:
have that on https://github.com/foundriesio/meta-lmp/pull/1444. fixed now